ui/revenuecatui/build.gradle

```diff
@@ -53,6 +62,8 @@ dependencies {
     implementation libs.androidx.lifecycle.viewmodel.compose
     implementation libs.coil.compose
     implementation libs.coil.svg
+    implementation libs.activity.compose
+    implementation libs.androidx.fragment.ktx
```

Comment:
I've been thinking about splitting all this into a different module, so we don't clutter the `ui:revenuecat` dependency for people not using activities/fragments. The idea would be to have a different dependency for people using activities fragments with all these codes and dependencies.

The down side is to further split into more modules, and we might need to expose some things like the `setOfferingId` in the `PaywallViewOptions`... But I think it might be ok?

Reply:
I would keep it in the same module, so we don't end up with many modules again. But it's true `androidx.activity:activity-compose` is 1.1 MB, which is not small compared to the 300 KB the billing client takes

Reply:
Is 1MB much these days? 😅

Reply:
Well, usually it's much less than that for devs minifying their apps. It's true that we're using a ton of dependencies in the revenuecatui module, so I wasn't sure about trying to optimize it a bit... But yeah, we can keep it in one module for now and avoid having a ton of modules 👍